package.json

```diff
@@ -64,6 +64,8 @@
     "build:account-export": "nx run-many --target=build --projects=account-export --configuration=production",
     "build:ramper-wallet": "nx run-many --target=build --projects=ramper-wallet --configuration=production",
     "build:near-mobile-wallet": "nx run-many --target=build --projects=near-mobile-wallet --configuration=production",
+    "examples:vanillajs:generateimportmap": "(cd examples/vanillajs && jspm link index.html -o index.html)",
+    "examples:vanillajs:serve": "http-server examples/vanillajs",
```

Comment:
Would it make sense to combine these two scripts (pretty difficult to remember them)?

Also, let's rename `examples:vanillajs:serve` to `serve:vanillajs` and move it down to the other scripts for serving angular and react examples.

I would suggest that we disable cache for this example by adding the option `-c-1` to the serve script because the browser is immediately caching `main.js` and changes are not being picked without hard-reload.